src/coord/timestamp.rs

```diff
@@ -27,6 +34,7 @@ use dataflow_types::{
 use expr::SourceInstanceId;
 
 use crate::coord;
+use chrono::Utc;
```

Comment:
Similar to comment below: proper sorting for these imports is to put both `use chrono::Utc` and the `rusoto` imports in the very top block, with the other third-party crates.